dashboard/app/models/workshop.rb

```diff
@@ -3,7 +3,7 @@ class Workshop < ActiveRecord::Base
 
   validates_inclusion_of :program_type, in: PROGRAM_TYPES, on: :create
   # A Workshop has multiple well defined Time Segments (eg. each morning/afternoon of a workshop is a separate time segment)
-  has_many :segments, dependent: :destroy
+  has_many :segments, dependent: :destroy, order: :start
```

Comment:
are segments still being sorted in js? That shouldn't be necessary now (if they are still being sorted in js, and if that sort is actually doing anything, it's possible that the attendances will be matched with the wrong segments).